tedana/decomposition/_utils.py

```diff
@@ -40,3 +41,51 @@ def eimask(dd, ees=None):
         imask[np.logical_and(m > lthr, m < hthr), ee] = True
 
     return imask
+
+
+def dwtmat(mmix):
+    """
+    Wavelet transform data.
+
+    Parameters
+    ----------
+    mmix : :obj:`numpy.ndarray`
+        Data to wavelet transform.
+
+    Returns
+    -------
+    mmix_wt : :obj:`numpy.ndarray`
+        Wavelet-transformed data.
+    cAlen : :obj:`int`
```

Comment:
Is there a reason to call it `cAlen` here and `cAl` in `idwtmat()` ?

Reply:
I just copied the functions from the old version of tedana and removed the `correct_size` argument in [`idwtmat`](https://github.com/ME-ICA/me-ica/blob/da6ac4c23a59145530a16bd6f25c676460fe9436/meica.libs/tedana.py#L480), which `pywt` must have dropped at some point.